modules/core/src/lib/mode-handlers/mode-handler.js

```diff
@@ -29,7 +30,7 @@ export type EditHandle = {
 export type EditAction = {
   updatedData: FeatureCollection,
   editType: string,
-  featureIndex: number,
+  featureIndex: number | Array<number>,
```

Comment:
I think it is strange to have `featureIndex` sometimes be a single number and sometimes an array. To support this and other edits in the future, we should probably rename to `featureIndexes` and always be an array. This would mean changing the existing modes to pass an array.

We also may want to do some future-proofing with `positionIndexes` and `position` as well.

How about this structure?
```
export type EditAction = {
  updatedData: FeatureCollection,
  editType: string,
  featureIndex: number[],
  editContext: ?any // For some edits this will contain positionIndexes, sometimes position. But nebula won't dictate the particular properties. This is can be fully dynamic and populated by the particular mode, or some mode in the future.
```

If we go this route, we should probably do it in its own PR and document the breaking change in the changelog.

Reply:
@supersonicclay I fully agree with you here. The reason I did this was to make as little impact to existing code as possible. I'll create a follow up pull request for the featureIndexes change.

Reply:
I actually think the tweak to `EditAction` should come *before* this PR. But I'm want to see if @lxe and/or @georgios-uber has concern with the breaking change first.

Reply:
When you mentioned moving positionIndexes and position into an editContext object, were you talking about just talking about the EditAction type, or did you also want changes made to the EditHandle type?